docs/large_inputs.md

```diff
@@ -1,63 +1,127 @@
+[`--worker_disk_type`](#--worker_disk_type)) when choosing a large (more than
+`n1-standard-8`) machine, which yields higher disk IOPS and can avoid idle CPU
+cycles. Note that disk is significantly cheaper than CPU, so always try to
+optimize for high CPU utilization rather than disk usage.
 
-## Disk size
+## `--disk_size_gb`
 
 By default, each worker has 250GB of disk. The aggregate amount of disk space
 from all workers must be at least as large as the uncompressed size of all VCF
 files being processed. However, to accomoddate for intermediate stages of the
 pipeline and also to account for the additional overhead introduced by the
 transforms (e.g. the sample name is repeated in every record in the BigQuery
 output rather than just being specified once as in the VCF header), you
-typically need 3 to 4 times the total size of the raw VCF files. You may change
-the disk size of every worker by adding `--disk_size_gb <disk size>` to the
-pipeline args.
+typically need 3 to 4 times the total size of the raw VCF files.
+
+In addition, if [merging](variant_merging.md) or
+[--num_bigquery_write_shards](#--num_bigquery_write_shards) is enabled, you may
+need more disk per worker (e.g. 500GB) as the same variants need to be
+aggregated together on one machine.
+
+## `--worker_disk_type`
+
+SSDs provide significantly more IOPS than standard persistent disks, but are
+more expensive. However, when choosing a large machine (e.g. `n1-standard-16`),
+they can reduce cost as they can avoid idle CPU cycles due to disk IOPS
+limitations.
+
+As a result, we recommend using SSDs if [merging](variant_merge.md) or
+[--num_bigquery_write_shards](#--num_bigquery_write_shards) is enabled: these
+operations require "shuffling" the data (i.e. redistributing the data among
+workers), which require significant disk I/O.
+
+Set
+`--worker_disk_type compute.googleapis.com/projects//zones//diskTypes/pd-ssd`
+to use SSDs.
+
+## `--num_bigquery_write_shards`
+
+Currently, the write operation to BigQuery in Dataflow is performed as a
+postprocessing step after the main transforms are done. As a workaround for
+BigQuery write limitations (more details
+[here](https://github.com/googlegenomics/gcp-variant-transforms/issues/199)),
+we have added "sharding" when writing to BigQuery. This makes the data load
+to BigQuery significantly faster as it parallelizes the process and enables
+loading large (>5TB) data to BigQuery at once.
+
+As a result, we recommend setting `--num_bigquery_write_shards 20` when loading
+any data that has more than 1 billion rows (after merging) or 1TB of final
+output. Note that using a larger value (e.g. 50) can cause BigQuery write to
+fail as there is a maximum limit on the number of concurrent writes per table.
+
+## `--partition_config_path`
 
-## Pipeline resource estimator tool
+Partitioning the output can save significant query costs once the data is in
+BigQuery. It can also optimize the pipeline (e.g. it natively shards the
```

Comment:
It can also optimize the cost/time of the pipeline

Reply:
Done.